scripts/cypress.js

```diff
@@ -16,10 +16,12 @@ debug('starting the CLI in dev mode with args %o', {
   args,
 })
 
-const exit = ({ code }) => {
-  process.exit(code)
-}
-
 execa(pathToCli, args, { stdio: 'inherit' })
-.then(exit)
-.catch(exit)
+.then(({ code }) => {
+  debug('exiting normally %o', { code })
```

Comment:
so ... if we got burnt by this once, let's not have it repeat it again. Imagine `execa` changes and ALSO does `code -> exitCode` for normal execution. Then this will again silently pass, even if `exitCode: 1`. 

```js
.then(({ code }) => {
  if (typeof code !== 'number') {
    console.error('Problem: missing exit code: code)
    process.exit(1)
  }
  // normal thing
})
```